main.js

```diff
@@ -19,49 +18,27 @@ Apify.main(async () => {
     if (!input) crash('Key-value store does not contain INPUT.');
     const { query } = input;
     if (!query || typeof query !== 'string') crash('Query loaded from INPUT is empty or not a string.');
-
-    const commonHeaders = {
-        'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36',
-    };
-
     let attempts = 20;
     let response;
     do {
```

Comment:
this loop could become a CheerioCrawler / BasicCrawler instance, the code is very much optimistic that things will just work (no blocks, proper json payload received, 200 http code, etc)